recipes/sqlcipher/all/conanfile.py

```diff
@@ -74,13 +108,15 @@ def _build_autotools(self):
         autotools_env = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
         if self.settings.os == "Linux":
             autotools_env.libs.append("dl")
+            if not self.options.with_largefile:
```

Comment:
```suggestion
            if not self.options.get_safe("with_largefile"):
```

Reply:
I don't think this is needed, since the option is deleted when `settings.os != "Linux"`